go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/bitnami-labs/sealed-secrets
 
-go 1.19
+go 1.20
```

Comment:
We don't really need to update this as this just indicates the minimum compatible version, but it's fine.